src/Common/src/Interop/Linux/Interop.Data.cs

```diff
@@ -4,8 +4,5 @@
 
 internal static partial class Interop
 {
-    internal static partial class Data
-    {
-        public const string Odbc32 = "libodbc.so.2";
-    }
+    public const string Odbc32 = "libodbc.so.2";
```

Comment:
I think what Steve meant is that we can actually move this into the Interop.Libraries file and delete the Interop.Data files as they only have this constant.

Reply:
Yup, sorry for not being clear.